tasks/set_acl_toolchains.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Give proper ACL permissions
+  acl:
+         path: "/var/results/toolchains"
+         entity: "{{ item }}"
+         etype: group
+         permissions: rx
```

Comment:
We don't want rx from users in here ever.

Reply:
Which is the problem, r or x ?

Reply:
d'oh